develop/toolkit/interoperability/paraspell-xcm-sdk/transfer-tokens-between-parachains.md

```diff
@@ -0,0 +1,183 @@
+```
+
+Do not execute it just yet. You will perform a dry run of this transaction first to ensure it works as expected.
+
+## Perform a Dry Run
+
+Dry runs simulate the transaction without broadcasting it, allowing you to confirm success in advance.
+
+Add the following dry run code to your `index.ts` script:
+
+```ts title="index.ts"
+--8<-- 'code/develop/toolkit/interoperability/paraspell/index.ts:49:70'
+```
+Go ahead and run the script.
+
+```bash
+bun run index.ts
+```
+
+The result of the dry run will be similar to this:
+
+--8<-- 'code/develop/toolkit/interoperability/paraspell/dry-run-output.html'
+
+## Verify the Existential Deposit
+
+Check if the recipient account meets the [Existential Deposit (ED)](/polkadot-protocol/glossary/#existential-deposit){target=\_blank} requirement before sending by using [`verifyEdOnDestination`](https://paraspell.github.io/docs/sdk/xcmUtils.html#verify-ed-on-destination){target=\_blank}:
+
+```ts title="index.ts"
+--8<-- 'code/develop/toolkit/interoperability/paraspell/index.ts:72:88'
+```
+Comment out the `dryRunTransfer()` and execute the `verifyED()` by running:
+
+```bash
+bun run index.ts
+```
+
+After that, you will get output confirming the ED:
+
+--8<-- 'code/develop/toolkit/interoperability/paraspell/ed-verification-output.html'
+
+## Get Transfer Info and Fee Estimates
+
+Before sending an XCM transaction, it is helpful to estimate the fees associated with executing and delivering the cross-chain message.
+
+ParaSpell has a helpful function for this: [`getTransferInfo()`](https://paraspell.github.io/docs/sdk/xcmUtils.html#xcm-transfer-info){target=\_blank}. This function returns an estimate of the associated XCM fees, along with the account's balance before and after the fees are paid.
+
+```ts title="index.ts"
+--8<-- 'code/develop/toolkit/interoperability/paraspell/index.ts:90:106'
+```
+
+Comment out the `verifyED()` and execute the `XcmTransferInfo()` by running:
+
+```bash
+bun run index.ts
+```
+
+You should be able to see all the information for your transfer:
+
+--8<-- 'code/develop/toolkit/interoperability/paraspell/transfer-info-output.html'
+
+Now that you have:
+
+- Completed a successful dry run of the transaction
+- Verified the existential deposit on the recipient account
+- Obtained an estimate of the associated XCM fees
+
+Now you can execute the transfer function by adding the following statement:
+
+Add the following code:
+
+```typescript title="index.ts"
+--8<-- 'code/develop/toolkit/interoperability/paraspell/index.ts:108:108'
+```
+
+Comment out the `XcmTransferInfo()` and execute the transfer:
+
+```bash
+bun run index.ts
+```
+
+Your `transfer` function will submit the transaction, and you will get the following output:
+
+--8<-- 'code/develop/toolkit/interoperability/paraspell/transfer-output.html'
+
+Once the transaction is successfully included in a block, you will see the recipient's account balance updated, and you will receive output similar to the one below.
```

Comment:
```suggestion
Once the transaction is successfully included in a block, you will see the recipient's account balance updated, and you will receive output similar to the following.
```
Words like "above" and "below' are less friendly to screen reader devices. "Previous" and "following" are preferred